src/analog/commands/uplink/getArchiveEvents.ts

```diff
@@ -0,0 +1,150 @@
+/**
+ * Command to receive events from device archive.
+ *
+ * @packageDocumentation
+ *
+ * @example create command instance from command body hex dump
+ * ```js
+ * import * as getArchiveEvents from 'jooby-codec/analog/commands/uplink/getArchiveEvents.js';
+ *
+ * // 1 event "MAGNET_ON" at 2023.04.05 13:17:20 GMT
+ * const bytes = [0x2b, 0xc0, 0x31, 0x60, 0x01, 0x01];
+ *
+ * // decoded payload
+ * const parameters = getArchiveEvents.fromBytes(bytes);
+ *
+ * console.log(parameters);
```

Comment:
such output is commented in other commands

shuold we uncomment it everywhere or comment here?

Reply:
we can do it like this:

 * console.log(parameters);
 * // output:
 * {
 *     eventList: [
 *         {
 *             time2000: 734015840,
 *             id: 1,
 *             sequenceNumber: 1
 *         }
 *     ]
 * }

Reply:
comments removed, console - already output, so 'output:' comment no necessary to add